src/Database/LSMTree/Internal/RawBytes.hs

```diff
@@ -208,6 +210,14 @@ instance Monoid RawBytes where
     mempty = coerce PV.empty
     mconcat = coerce PV.concat
 
+-- | O(n) Yield the argument, but force it not to retain any extra memory,
+-- possibly by copying it.
```

Comment:
Looks like `PV.force` always copies, doesn't it?

Reply:
It's the same comment as on `PV.force`, but yes I suppose it always copies

Reply:
In `bytestring` this is called `copy` which I think is a better name. Force sounds like WHNF.

Reply:
I'll call it copy instead